packages/bits/src/lib/toolbar/toolbar-keyboard.service.ts

```diff
@@ -0,0 +1,47 @@
+import { Injectable } from "@angular/core";
+import { KEYBOARD_CODE } from "../../constants";
+
+@Injectable()
```

Comment:
Does this need a `{ providedIn: "root" }` so that consumers don't need to mention it in their module providers arrays?

Reply:
nevermind, I just realized that each toolbar component has its very own instance of the service

Reply:
we inject this service in component scope, not module.